app/components/admin/design_header_component.html.erb

```diff
@@ -0,0 +1,45 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<% helpers.html_title t(:label_administration), t(:label_custom_style), @title %>
+<%=
+  render(Primer::OpenProject::PageHeader.new) do |header|
+    header.with_title { t(:label_custom_style) }
+    header.with_breadcrumbs([{ href: admin_index_path, text: t(:label_administration) },
+                             t(:label_custom_style)])
+    header.with_description { t(:label_custom_style_description) }
+    header.with_tab_nav(label: nil) do |tab_nav|
+      @tabs.each do |tab|
+        tab_nav.with_tab(selected: selected_tab(@tabs) == tab, href: tab[:path]) do |t|
+          t.with_text { tab[:label] }
+        end
+      end
+    end if @tabs.present?
+  end
```

Comment:
Nitpick: I'd suggest to use an explicit if-block instead of a trailing-if when your happy execution-path spans over more than one line. It makes the body of the if-clause much more readable. Also, I prefer to give the code at least some breathing room by adding an empty line before a bigger code chunk. Combined, my suggestion would look like this: what do you think?

```suggestion
  render(Primer::OpenProject::PageHeader.new) do |header|
    header.with_title { t(:label_custom_style) }
    header.with_breadcrumbs([{ href: admin_index_path, text: t(:label_administration) },
                             t(:label_custom_style)])
    header.with_description { t(:label_custom_style_description) }
    
    if @tabs.present?
      header.with_tab_nav(label: nil) do |tab_nav|
        @tabs.each do |tab|
          tab_nav.with_tab(selected: selected_tab(@tabs) == tab, href: tab[:path]) do |t|
            t.with_text { tab[:label] }
          end
        end
      end
  end
```

Reply:
yeah, your suggestion is more readable 👍